README.md

```diff
@@ -66,7 +67,7 @@ const agent: Luminati = luminati.changeIp({ sessionId });
 
 ### Strategy: Change ip every requests
```

Comment:
Do we want an example implementation specific to shifter.io?
We would also have it as a little github wiki?

Reply:
Could be useful for everyone.